drivers/sensor/adxl362/adxl362.c

```diff
@@ -1,4 +1,6 @@
 /* adxl362.c - ADXL362 Three-Axis Digital Accelerometers */
+
+#define DT_DRV_COMPAT adi_adxl362
```

Comment:
move this below the license/copyright header

Reply:
done